ci-operator/step-registry/hypershift/kubevirt/create/hypershift-kubevirt-create-commands.sh

```
@@ -237,4 +259,120 @@ oc wait --timeout=30m --for=condition=Available --namespace=${CLUSTER_NAMESPACE_
 echo "Cluster became available, creating kubeconfig"
 $HCP_CLI create kubeconfig --namespace="${CLUSTER_NAMESPACE_PREFIX}" --name="${CLUSTER_NAME}" >"${SHARED_DIR}/nested_kubeconfig"
 
+# Post-creation localnet setup: DHCP workaround, port security clearing, ipecho deployment
+if [[ "${ATTACH_DEFAULT_NETWORK}" == "localnet" ]]; then
+  LOCALNET_SUBNET="${LOCALNET_SUBNET:-192.168.111.0/24}"
+  # Derive gateway IP (.1) from the subnet
+  LOCALNET_GW=$(echo "${LOCALNET_SUBNET}" | sed 's|\.[0-9]*/|.1|')
```

Shellcheck (0.11.0)

[style] See if you can use ${variable//search/replace} instead.

(SC2001)